src/odemis/driver/test/simstreakcam_test.py

```diff
@@ -60,7 +58,8 @@ class TestSimStreakCamGenericCam(VirtualTestCam, unittest.TestCase):
     camera_type = CLASS_STREAKCAM
     camera_kwargs = KWARGS_STREAKCAM
 
-    def setUp(self):
```

Comment:
Wouldn't just adding the line in `def setUp(self):` work?

`super().setUp()`

Reply:
yes, but there is not really a reason to have this in the setUp, or is there? The advantage of having it in the setUpClass is that it only gets called once.